geonode/local_settings.py.qgis.sample

```diff
@@ -29,10 +29,15 @@ try:
 except ImportError:
     pass
 
-# GEONODE_APPS += ("geonode.qgis_server", )
+qgis_server_apps_package = "geonode.qgis_server"
+
 GEONODE_APPS = list(GEONODE_APPS)
-# INSTALLED_APPS += ("geonode.qgis_server", )
+if qgis_server_apps_package not in GEONODE_APPS:
```

Comment:
We now only include `geonode.qgis_server` if it is not included in  modified settings file.